### PR TITLE
Feat/automized issuer verification

### PR DESCRIPTION
**Pull request by YouQam**
This PR optimize the process of issuer validation by checking whether email domain matches institution website domain.

### CODE REVIEW

File: apps/mainsite/utils.py [hunk 1]
```diff
@@ -485,3 +485,7 @@ def convert_svg_to_png(svg_string, height, width):
     except ValueError:
         # Issuing decoding response JSON
         return False
+
+def verifyIssuerAutomatically(url, email):
+    emailDomain = email.split('@')[1]
+    return emailDomain in url
```
Comment (timber-they) [inline apps/mainsite/utils.py:491]: This might cause wrong results if the email domain is merely a substring of the issuer domain

**Review by timber-they — CHANGES_REQUESTED** [review]

File: apps/mainsite/utils.py (hunk 1 above)
Comment (YouQam) [reply apps/mainsite/utils.py:491]: Can you provide an example where it might cause a wrong result?

Comment (YouQam) [reply apps/mainsite/utils.py]: I updated the implementation of extracting URL domain and matching domains.

**Review by timber-they — APPROVED** [review]
Sorry I didn't see your previous comment! This should be fine though

**Pull request merged**